src/app/containers/Brand/index.jsx

```diff
@@ -3,9 +3,26 @@ import Brand from '@bbc/psammead-brand';
 import { ServiceContext } from '../../contexts/ServiceContext';
 
 const BrandContainer = () => {
-  const { brandName, brandSVG } = useContext(ServiceContext);
+  const { product, serviceLocalizedName, brandSVG, frontpageUrl } = useContext(
+    ServiceContext,
+  );
+  const svgMaxHeight = 24;
+  const svgMinHeight = 16;
+  const svgRatio = brandSVG && brandSVG.ratio;
+  const minWidth = svgRatio * svgMinHeight;
+  const maxWidth = svgRatio * svgMaxHeight;
 
-  return <Brand brandName={brandName} svg={brandSVG} />;
+  return (
+    <Brand
```

Comment:
Need to pass `borderTop` or `borderBottom` depending on if its in the header or footer, passed as a prop to the brand container